nf_core/schema.py

```diff
@@ -62,15 +66,20 @@ def get_schema_path(self, path, local_only=False, revision=None):
 
         # Path does not exist - assume a name of a remote workflow
         elif not local_only:
-            self.pipeline_dir = nf_core.list.get_local_wf(path, revision=revision)
-            self.schema_filename = Path(self.pipeline_dir, "nextflow_schema.json")
+            self.schema_filename = path / "nextflow_schema.json"
```

Comment:
If we don't pull the pipeline with `nf-core.list.get_local_wf` the `self.schema_filename` path doesn't exist. It fails [later](https://github.com/nf-core/tools/pull/2891/files#diff-d2e52df2ef64bc68c00bdcd2c39861b00e5d519ea8409b8ed3366c488378f516R115) in `load_schema()`

I have tested this functionality by creating a `params.json` file and using `nf-core schema validate rnaseq params.json`

Reply:
hmm, it just seemed to me `self.pipeline_dir` was `None` in the test, but maybe I just replace it with an empty string then...